qujax/__init__.py

```diff
@@ -1,3 +1,8 @@
+"""
```

Comment:
Do you want to add a copyright note to all the files at some time?

Reply:
probably a good idea yeah